src/linear_algebra/projective_space/independence.lean

```diff
@@ -0,0 +1,127 @@
+/-
+Copyright (c) 2022 Michael Blyth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Blyth
+-/
+
+import linear_algebra.projective_space.basic
+
+/-!
+# Independence in Projective Space
+
+In this file we define independence and dependence of families of elements in projective space.
+
+## Implementation Details
+
+We use an inductive definition to define the independence of points in projective
+space, where the only constructor assumes an independent family of vectors from the
+ambient vector space. Similarly for the definition of dependence.
+
+## Results
+
+- A family of elements is dependent if and only if it is not independent.
+- Two elements are dependent if and only if they are equal.
+
+# Future Work
+
+- Define collinearity in projective space.
+- Prove the axioms of a projective geometry are satisfied by the dependence relation.
+- Define projective linear subspaces.
+-/
+
+
+variables {K V : Type*} [field K] [add_comm_group V] [module K V]
```

Comment:
If we add `ι` and `f` as a `variables` then we can drop them from all the statements below.
```suggestion
variables {ι K V : Type*} [field K] [add_comm_group V] [module K V] {f : ι → ℙ K V}
```
Note also that since all lemmas about `f` are iff lemmas, it's actually more desirable to have `f` consistently as an implicit variable.